packages/compartment-mapper/b.js

```diff
@@ -0,0 +1,347 @@
+'use strict';
```

Comment:
Was this file intentionally checked-in?

Reply:
yes, but temporarily, as the visualization of what progress/changes we're making to an example bundle.

it was very useful for understanding changes without having to run/debug the code.